Servers/Administration/OJS.Servers.Administration/Controllers/ContestCategoriesController.cs

```diff
@@ -116,7 +122,7 @@ protected override Task BeforeEntitySaveOnCreateAsync(ContestCategory entity, Ad
 
     private async Task CascadeDeleteCategories(ContestCategory contestCategory)
     {
-        foreach (var children in contestCategory.Children.ToList())
+        foreach (var children in contestCategory.Children)
```

Comment:
Can we use `contestCategory.Children.forEach()` instead of foreach statement? It's better to use it for more readability and project practice.